src/components/molecules/Visualizer/Plugin/api.ts

```diff
@@ -156,12 +160,18 @@ export function commonReearth({
         get position() {
           return camera();
         },
+        get viewport() {
+          return viewport;
```

Comment:
The plugin API often uses getters, so make sure to return the value directly instead of returning a function.

```suggestion
          return viewport();
```